sources/PanelButton.cpp

```diff
@@ -68,6 +73,14 @@ PanelButton::CreatePanel(int8 which, const BView* target, uint32 flavors,
 		panel = new BFilePanel(B_OPEN_PANEL, &msgr, NULL, flavors, false, NULL,
 			filter);
 
-		panel->Window()->SetTitle(pairs[which].locale);
+		BString title = B_TRANSLATE("Filer");
+		title += ": ";
+
+		if (prefix != NULL)
+			title << prefix << ' ';
```

Comment:
> For some reason, there are two spaces after the prefix.

I forgot I already added a space for non-null `prefix` in `PanelButton::CreatePanel`, which is the right way to do it, so I should not add the space again in `TestView`.

> We don't know where the 'pairs[which].locale' appears in other languages.

I don't understand. `paris[which].locale` should refer to the corresponding, localized string. Does it not work as expected?